### PR TITLE
ENG-4264 404 not found on load and CORS configuration update

### PR DESCRIPTION
**Pull request by alepintus**

### CODE REVIEW

File: src/main/java/com/entando/template/config/SecurityConfig.java [hunk 1]
```diff
@@ -50,6 +53,19 @@ protected SessionAuthenticationStrategy sessionAuthenticationStrategy() {
...
     }
 
+    @Profile("development")
+    @Bean
+    public WebMvcConfigurer corsConfigurer() {
+        return new WebMvcConfigurer() {
+            @Override
+            public void addCorsMappings(CorsRegistry registry) {
+                registry.addMapping("/**")
+                        .allowedOrigins("http://localhost:3000")
```
Comment (cecchisandrone) [inline src/main/java/com/entando/template/config/SecurityConfig.java:63]: can we allow just localhost without specific port?

File: src/main/java/com/entando/template/rest/EntTemplateController.java [hunk 2]
```diff
@@ -79,7 +78,6 @@ public PagedContent<TemplateResponseView, EntTemplate> getFilteredTemplates(
 
 	@Operation(summary = "Get the template details by id", description = "Private api, authentication required.")
```
Comment (zonia3000) [inline src/main/java/com/entando/template/rest/EntTemplateController.java]: This one is public, I would replace that misleading description

Comment (alepintus) [reply src/main/java/com/entando/template/rest/EntTemplateController.java]: done, description removed

File: src/main/java/com/entando/template/config/SecurityConfig.java (hunk 1 above)
Comment (alepintus) [reply src/main/java/com/entando/template/config/SecurityConfig.java:63]: I just did a test  and replacing the port with * character we have the CORS error on the  API request,  so we have to keep the 3000 port on the allowed origins string

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**